lib/vty.c

```diff
@@ -390,6 +390,21 @@ int vty_json_no_pretty(struct vty *vty, struct json_object *json)
 	return vty_json_helper(vty, json, JSON_C_TO_STRING_NOSLASHESCAPE);
 }
 
+
+void vty_json_key(struct vty *vty, const char *key, bool *first_key)
+{
+	vty_out(vty, "%s\"%s\":", *first_key ? "{" : ",", key);
+	*first_key = false;
+}
+
+void vty_json_close(struct vty *vty, bool seen_key)
+{
+	if (seen_key)
```

Comment:
This name sounds like a reverse to me. If _seen_, then why to open it? `first_key` looks would be just a better one?

Reply:
you are correct. Changed